src/components/version-redirect.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Redirect} from 'react-router-dom';
+import Loading from './loading';
+
+export default class VersionRedirect extends React.Component {
+  constructor (props) {
+    super (props);
+    this.state = {
+      pageId: null,
+      error: null
+    };
+  }
+
+  componentDidMount () {
+    const versionId = this.props.match.params.versionId;
+    this.props.api.getVersionById(versionId)
+      .then(data => this.setState({pageId: data.page_uuid}))
+      .catch(error => this.setState({error: error}));
+  }
+
+  render () {
+    if (this.state.error) {
+      return (
+        <p className="alert alert-danger" role="alert">
+          Error: We couldn't find the version you're looking for.
+          Please check you provided the correct versionID or alert the dev team.
```

Comment:
I don’t think “or alert the dev team” makes sense in the larger context if someone else redeploys this or as we add more external users in the future. Any EDGI analysts know they can just bug us on Slack.